### PR TITLE
Added rule for disallowing font-size less 15px

### PR DESCRIPTION
**Pull request by igorkamyshev**
Please, review my PR and give me feedback.

### CODE REVIEW

File: src/rules/font-size-large-enough/__tests__/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import rule, { messages, ruleName } from '../index';
+
+testRule(rule, {
+  ruleName,
+  config: [true],
+
+  accept: [
+    {
+      code: '.foo { font-size: 15px; }',
```
Comment (evilebottnawi) [inline src/rules/font-size-large-enough/__tests__/index.js]: Let's add test with UPPERCASE (`FONT-SIZE: 15PX;`)

File: src/rules/font-size-large-enough/index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  }
+
+  const result = node.nodes.some(
+    o =>
+      o.type === 'decl' &&
+      o.prop.toLowerCase() === 'font-size' &&
+      (!o.value.toLowerCase().endsWith('px') || parseFloat(o.value) >= 15)
+  );
+
+  return result;
```
Comment (YozhikM) [inline src/rules/font-size-large-enough/index.js]: Just return `node.nodes.some(...)` here

**Comment by YozhikM** [comment]
@igorkamyshev Thank you for PR!. It looks great.

@evilebottnawi Do we need to calculate the font size if it was not specified in `px`?

**Comment by igorkamyshev** [comment]
Thanks for your feedback! I fix issues and will return later.

File: src/rules/font-size-large-enough/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# font-size-large-enough
```
[the remark this answers is not in the document]
Comment (igorkamyshev) [reply src/rules/font-size-large-enough/README.md]: What about "font-size-is-readable"?

**Comment by evilebottnawi** [comment]
@YozhikM i think yes

**Comment by YozhikM** [comment]
@igorkamyshev Please, add new rule to main README too

**Comment by YozhikM** [comment]
I think we can merge the PR as is. In the event that community accepts this rule, we will make it smarter

**Comment by YozhikM** [comment]
@evilebottnawi Do you still have any comments?

**Comment by YozhikM** [comment]
@igorkamyshev  @evilebottnawi 
Thank you all for your help. Today I will publish a new version of the library in npm. We did a good job together!

**Pull request merged**